spec/integration/models/process_spec.rb

```diff
@@ -11,12 +11,22 @@
     @rest_client = ConnectionHelper.create_default_connection
     @suffix = AppstoreProjectHelper.suffix
     @project = ProjectHelper.get_default_project client: @rest_client
+    @archive_location = './spec/data/cc'
+    @options = { project: @project, client: @rest_client }
   end
 
   after(:all) do
     @rest_client.disconnect
   end
 
+  def create_process
+    GoodData::Process.deploy(@archive_location, @options.merge(name: 'Test process GRAPH'))
+  end
+
+  def destroy_process(process)
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)